pkg/prometheus/agent/statefulset_test.go

```diff
@@ -181,11 +170,8 @@ func TestWALCompression(t *testing.T) {
 		}
 
 		if found != test.shouldContain {
-			if test.shouldContain {
-				t.Fatalf("expected Prometheus args to contain %v, but got %v", test.expectedArg, promArgs)
-			} else {
-				t.Fatalf("expected Prometheus args to NOT contain %v, but got %v", test.expectedArg, promArgs)
-			}
+			require.False(t, test.shouldContain, "expected Prometheus args to contain %v, but got %v", test.expectedArg, promArgs)
+			require.True(t, test.shouldContain, "expected Prometheus args to NOT contain %v, but got %v", test.expectedArg, promArgs)
```

Comment:
I feel this is not a good method. There are many other places with same code. Wdyt?